cmd/thanos/receive.go

```diff
@@ -28,10 +32,7 @@ func registerReceive(m map[string]setupFunc, app *kingpin.Application, name stri
 	cmd := app.Command(name, "Accept Prometheus remote write API requests and write to local tsdb (EXPERIMENTAL, this may change drastically without notice)")
 
 	grpcBindAddr, cert, key, clientCA := regGRPCFlags(cmd)
-	httpMetricsBindAddr := regHTTPAddrFlag(cmd)
```

Comment:
still changes are there for receive, cc @brancz  not sure if you have seen those! (:

Reply:
They were discussed on [slack thread](https://improbable-eng.slack.com/archives/CADR0UWS2/p1553239740260200) so hopefully it's ok